mava/components/jax/executing/base.py

```diff
@@ -46,7 +46,7 @@ def on_execution_init_start(self, executor: SystemExecutor) -> None:
         Returns:
             None.
         """
-        executor._interval = self.config.interval  # type: ignore
+        executor.store._interval = self.config.interval  # type: ignore
```

Comment:
I do not think that this variable should have an _ indicating private variable as it is part of the store and will be accessible

Reply:
Agreed. I updated the code :+1: